hazelcast/src/test/java/com/hazelcast/client/impl/MemberImplTest.java

```diff
@@ -41,15 +42,15 @@ public void testConstructor() {
 
     @Test
     public void testConstructor_withAddress() {
-        MemberImpl member = new MemberImpl(address);
+        MemberImpl member = new MemberImpl(address, Version.of("3.8.0"));
```

Comment:
In the particular test we need a fixed version in the constructed member objects to validate. In any case I have reverted to using a `Version` object for the current codebase version in other tests which do not need a fixed version to validate, to avoid unpleasant surprises in the future.